viscy/data/triplet.py

```diff
@@ -404,7 +403,13 @@ def __init__(
             Whether to pin memory in CPU for faster GPU transfer, by default False
         z_window_size : int, optional
             Size of the final Z window, by default None (inferred from z_range)
+        cache_pool_bytes : int, optional
+            Size of the per-process tensorstore cache pool in bytes, by default 0
         """
+        if num_workers > 1:
+            warnings.warn(
+                "Using more than 1 thread worker will likely degrade performance."
```

Comment:
We should add something more informative of why either in the docstrings or here. I prefer the docstrings if we are using batchedtransforms

Reply:
and we are not aiming for backwards compatibility then rename the whole argument

Reply:
Added to docstring.